apps/docs/content/guides/database/custom-postgres-config.mdx

```diff
@@ -188,5 +190,6 @@ alter role "<role_name>" set "<setting_name>" to default;
 ### Considerations
 
 1. Changes through the CLI might restart the database causing momentary disruption to existing database connections; in most cases this should not take more than a few seconds. However, you can use the --no-restart flag to bypass the restart and keep the connections intact. Keep in mind that this depends on the specific configuration changes you're making. if the change requires a restart, using the --no-restart flag will prevent the restart but you won't see those changes take effect until a restart is manually triggered.
+1. If you use the --no-restart flag, particularly where read replicas exist, and later one of the database is restarted, this might lead to a conflict of mismatching parameters and prevent one of the database coming up correctly.
```

Comment:
small typo

```suggestion
1. If you use the --no-restart flag, particularly where read replicas exist, and later one of the databases is restarted, this might lead to a conflict of mismatching parameters and prevent one of the databases coming up correctly.
```